src/rdd/rdd.rs

```diff
@@ -992,6 +994,68 @@ pub trait Rdd: RddBase + 'static {
         let min_fn = Fn!(|x: Self::Item, y: Self::Item| x.min(y));
         self.reduce(min_fn)
     }
+
+    fn top(&self, num: usize) -> Result<Vec<Self::Item>>
+    where
+        Self: Sized,
+        Self::Item: Data + Ord,
```

Comment:
Not sure if the `Ord` should be `PartialOrd` bound, to handle RDD with float numbers. Same for the below `take_ordered()`

Reply:
We can take a `PartialOrd` here yes.

Reply:
Yup PartialOrd is enough here

Reply:
Looks like BinaryHeap requires `Ord` trait